lib/athens/version.rb

```diff
@@ -1,3 +1,3 @@
 module Athens
-  VERSION = "0.3.3"
+  VERSION = "0.3.4"
```

Comment:
You don't need to do this, I'll take care of bumping the version and updating the changelog on my end when I release.